wgpu/src/backend/web.rs

```diff
@@ -1090,9 +1146,24 @@ impl crate::context::Context for Context {
             max_uniform_buffer_binding_size: limits.max_uniform_buffer_binding_size() as u32,
             max_storage_buffer_binding_size: limits.max_storage_buffer_binding_size() as u32,
             max_vertex_buffers: limits.max_vertex_buffers(),
+            max_buffer_size: limits.max_buffer_size() as u64,
             max_vertex_attributes: limits.max_vertex_attributes(),
             max_vertex_buffer_array_stride: limits.max_vertex_buffer_array_stride(),
+            min_uniform_buffer_offset_alignment: limits.min_uniform_buffer_offset_alignment(),
+            min_storage_buffer_offset_alignment: limits.min_storage_buffer_offset_alignment(),
+            max_inter_stage_shader_components: limits.max_inter_stage_shader_components(),
+            max_compute_workgroup_storage_size: limits.max_compute_workgroup_storage_size(),
+            max_compute_invocations_per_workgroup: limits.max_compute_invocations_per_workgroup(),
+            max_compute_workgroup_size_x: limits.max_compute_workgroup_size_x(),
+            max_compute_workgroup_size_y: limits.max_compute_workgroup_size_y(),
+            max_compute_workgroup_size_z: limits.max_compute_workgroup_size_z(),
+            max_compute_workgroups_per_dimension: limits.max_compute_workgroups_per_dimension(),
             ..wgt::Limits::default()
+            // The following are not part of WebGPU
+            /*
+              max_push_constant_size: limits.max_push_constant_size(),
+              max_non_sampler_bindings: limits.max_non_sampler_bindings(),
+            */
```

Comment:
Could we explicitly fill all of these out, so that future additions will be dealt with properly.